spec/system/planning_applications/requesting_changes_spec.rb

```diff
@@ -81,7 +81,7 @@
     expect(page).to have_content("Request for approval of changes to description")
     expect(page).to have_content("Application number: #{planning_application.reference}")
     expect(page).to have_content("At: #{planning_application.full_address}")
-    expect(page).to have_content("Request sent: #{description_change_request.created_at.strftime('%e %B %Y')}")
+    expect(page).to have_content("Request sent: #{description_change_request.created_at.strftime('%-e %B %Y')}")
```

Comment:
I don't understand what this commit is doing?

Reply:
It's an unrelated fix, which is why it's a separate commit. It changes %e to %-e to remove the leading padding on this date. Otherwise, if it's an application that was created in the first 10 days of the month, the spec fails because it's not expecting an extra space here. It has only just started failing because the spec was written when the relevant dates were all double-digit